chore: update references to old versions of the host (#208)
wasmCloud/wasmCloud.com: docs/developer/debugging/host.md

Host troubleshooting largely revolves around checking host logs and determining the issue from there. With wasmCloud we aim to log relevant information so that you can diagnose the source of the problem.

> The primary wasmCloud host runtime is the [wasmCloud](https://github.com/wasmCloud/wasmCloud/tree/main/crates/host) host, and this is what you'll be using if you're running the host from the release tarball or with the [wasmcloud](https://hub.docker.com/r/wasmcloud/wasmcloud) docker image. The Javascript host has different log messages and it's recommended to create an issue [here](https://github.com/wasmCloud/wasmcloud-js/issues/new) if you are running into errors in the browser.

Reviewer: ```suggestion
The primary wasmCloud host runtime is [`wasmcloud-host`](https://github.com/wasmCloud/wasmCloud/tree/main/crates/host), which you'll be using if you're running the host from the release tarball or with the [wasmcloud](https://hub.docker.com/r/wasmcloud/wasmcloud) docker image. The Javascript host has different log messages and it's recommended to create an issue [here](https://github.com/wasmCloud/wasmcloud-js/issues/new) if you are running into errors in the browser.
```
